src/pydata_sphinx_theme/assets/styles/content/_admonitions.scss

```diff
@@ -191,11 +191,33 @@ div.admonition,
     float: right;
     clear: both;
     margin-left: 0.5rem;
+    margin-top: 0;
 
     // Undo the .sidebar directive border
-    border: none;
+    border-width: 0 0 0 0.2rem;
 
-    // No margin since we have less horizontal space w/ the sidebar
+    &.attention,
```

Comment:
does this not inherit the border color from other CSS rules? I would have thought that happened. It feels like we are not being DRY by having this duplicated in both, but it's not a huge deal. Just want to check that these are definitely necessary

Reply:
indeed it does not. Without this, the border is white.

Reply:
I should add that I also expected it to inherit the correct color, and I'm not sure why it doesn't. Clearly I have not yet snatched the pebble from the CSS master's hand.

Reply:
what version of docutils are you using? I just tried cloning this PR locally and deleting all of these sections, and the colors showed up as usual in the sidebar areas, so maybe there's a docutils difference here and we should figure out if that's a regression?

Reply:
I'm on 0.18.1

Reply:
(but I should have thought to check that... sorry)

Reply:
weird, I just tried on Sphinx 5 and docutils 0.18.1 and it still worked fine

Reply:
maybe it's different in the `nox` docs-live environment?  Though I'm having difficulty figuring out how to list versions in that env...  it shows up without a name for me:

```
$ mamba env list
# conda environments:
#
base                     /opt/miniforge3
mnedev                *  /opt/miniforge3/envs/mnedev
                         /opt/pydata-sphinx-theme/.nox/docs-live  # <----- NO NAME?
```

Reply:
OK, i figured it out. my `docs-live` env has sphinx 4.2.0 and docutils 0.17.1

Reply:
that is probably why I was sometimes seeing different results when building the PST documentation vs building MNE-Python docs... they were building under different sphinx/docutils versions (meaning maybe some of the HTML tags/classes were actually different?)

Reply:
I would think that we want to still support sphinx 4 / docutils 0.17 (and thus if we need extra rules to make it look the same under sphinx 4 vs sphinx 5, then we should add those extra rules).  That is, until sphinx 5 is at least 6 months old. WDYT?

Reply:
yeah that makes sense to me - maybe we can just add a comment there like `// TODO: check that these rules are still needed after we deprecate Sphinx 4` ?

Reply:
done in #848